specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -564,621 +715,1772 @@
+        "operationId": "Weather_GetAirQualityHourlyForecast",
+        "x-ms-examples": {
+          "Get Hourly Air Quality Forecast": {
+            "$ref": "./examples/GetHourlyAirQuality.json"
+          }
+        },
+        "parameters": [
           {
-            "value": 6,
-            "name": "Kilometer",
-            "description": "kilometer"
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
           },
           {
-            "value": 7,
-            "name": "KilometersPerHour",
-            "description": "kilometersPerHour"
+            "$ref": "#/parameters/ApiVersion"
           },
           {
-            "value": 8,
-            "name": "Knots",
-            "description": "knots"
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/JsonFormat"
           },
           {
-            "value": 9,
-            "name": "MilesPerHour",
-            "description": "milesPerHour"
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/Language"
           },
           {
-            "value": 10,
-            "name": "MetersPerSecond",
-            "description": "metersPerSecond"
+            "name": "query",
+            "description": "Coordinates of the location for which air quality is requested. The applicable query is specified as a comma separated string composed by latitude followed by longitude e.g. \"47.641268,-122.125679\".",
+            "in": "query",
+            "required": true,
+            "x-ms-client-name": "coordinates",
+            "x-ms-parameter-location": "method",
+            "type": "array",
+            "collectionFormat": "csv",
+            "minItems": 2,
+            "maxItems": 2,
+            "items": {
+              "type": "number",
+              "format": "double"
+            }
           },
           {
-            "value": 11,
-            "name": "HectoPascals",
-            "description": "hectoPascals"
+            "name": "duration",
+            "description": "Specifies for how many hours from now we would like to know about the air quality. Available values are 1, 12, 24, 48, 72, 96. Default value is 1 hour.",
+            "default": 1,
+            "in": "query",
+            "type": "integer",
+            "format": "int32",
+            "enum": [
+              1,
+              12,
+              24,
+              48,
+              72,
+              96
+            ],
+            "x-ms-enum": {
+              "name": "HourlyDuration",
+              "modelAsString": true,
+              "values": [
+                {
+                  "value": 1,
+                  "description": "1 Hours"
+                },
+                {
+                  "value": 12,
+                  "description": "12 Hours"
+                },
+                {
+                  "value": 24,
+                  "description": "24 Hours"
+                },
+                {
```

Comment:
Not sure how these will translate to enum properties. Pretty sure they can't just be numbers in most languages.

Reply:
Thanks for the suggestion!
I have added name for all those enums.
Is it ok for you?